setup.cfg

```diff
@@ -46,6 +46,8 @@ install_requires =
     mccabe>=0.6,<0.7
     toml>=0.7.1
     colorama;sys_platform=="win32"
+setup_requires =
+  setuptools_scm
```

Comment:
According to the doc, moving this here to the `pyproject.toml` file would be the preferred. https://github.com/pypa/setuptools_scm#pyprojecttoml-usage

Reply:
Yeah, but we don't have ``pyproject.toml`` yet :D

Reply:
True 🤷🏻‍♂️